ietf/doc/utils_search.py

```diff
@@ -143,14 +143,15 @@ def fill_in_document_table_attributes(docs, have_telechat_date=False):
     xed_by = RelatedDocument.objects.filter(target__name__in=list(rfc_aliases.values()),
                                             relationship__in=("obs", "updates")).select_related('target')
     rel_rfc_aliases = dict([ (a.document.id, re.sub(r"rfc(\d+)", r"RFC \1", a.name, flags=re.IGNORECASE)) for a in DocAlias.objects.filter(name__startswith="rfc", docs__id__in=[rel.source_id for rel in xed_by]) ])
+    l = []
```

Comment:
This doesn't work. l is a temporary that is switched between two things inside the loop below. The assignment here is ignored, and the sort as written happens on whatever list l got assigned to last (leaving the other unsorted).